sql_playground/playground.txt

```diff
@@ -1,7 +1,23 @@
 You are now connected to database "nc_news_test" as user "dannyclark90".
- comment_id |                            body                             | article_id |    author     | votes |         created_at         
-------------+-------------------------------------------------------------+------------+---------------+-------+----------------------------
-         21 | Maybe, like a cat you've seen something minute on the wall. |         11 | butter_bridge |     0 | 2023-11-23 06:45:28.292208
-(1 row)
+DELETE 1
+ comment_id |                                                                                         body                                                                                         | article_id |    author     | votes |     created_at      
```

Comment:
please .gitignore this file, it's not necessary for the API, just for you to play with locally